### ISSUE TITLE
Opioid use disorder

### ISSUE THREAD

**Issue by hfish100** [obophenotype/human-phenotype-ontology#10272]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@hfish100 
This is [Addictive opioid use HP:0033515](https://hpo.jax.org/app/browse/term/HP:0033515)
